### ISSUE TITLE
experimentalWebKitSupport fails test when experimentalSessionAndOrigin is enabled without clear error

### ISSUE THREAD

**Issue by Light07** [cypress-io/cypress#23865]
### Current behavior
Tried to upgrade to Cypress 10.8.0, enable experimentalWebKitSupport and doing my test against webkit.

I used a customized folder structure as below:
<img width="310" alt="image" src="https://user-images.githubusercontent.com/9669013/190898997-91f221d1-0cc5-4ae1-9c60-efcd055d1e8f.png">

When i run test cases, it always failed with below error:
<img width="505" alt="image" src="https://user-images.githubusercontent.com/9669013/190899065-6573df16-4ea1-4ecd-8705-12b8619a9d63.png">

However, My tests run successfully on Chrome, it's may some thing related to customized folder structure.

The contents of **cypress.config.js** are as below:
```js
const { defineConfig } = require("cypress");

const browserify = require('@cypress/browserify-preprocessor')
const fs = require('fs-extra')
const path = require('path')


// add environment switch support
function getConfigurationByFile (file) {
  const pathToConfigFile = path.resolve(process.cwd(), '././envs', `cypress.${file}.json`)
  return fs.readJson(pathToConfigFile)
}



module.exports = defineConfig({
  e2e: {
    baseUrl: null,
    "specPattern": "src/tests/**/*.cy.js",
    "excludeSpecPattern":  [
      '**/snapshots/*',
      '**/__image_snapshots__/*',
    ],
    "fixturesFolder": "src/configs",
    "supportFile": "src/support/e2e.js",
    "experimentalSessionAndOrigin":true,
    setupNodeEvents(on, config) {
      require('cypress-failed-log/on')(on);
      const options = browserify.defaultOptions
      on('file:preprocessor', browserify(options))
      on('before:browser:launch', (browser = {}, launchOptions) => {
        launchOptions.args = require('cypress-log-to-output').browserLaunchHandler(browser, launchOptions.args);
        if (browser.name === 'chrome') {
          launchOptions.args.push('--auto-open-devtools-for-tabs')
          launchOptions.args.push('--disable-gpu');
          launchOptions.args.push('--disable-dev-shm-usage');
          return launchOptions;
        }
        return launchOptions;
      })

      let file = config.env.testEnv || 'dev'

      return getConfigurationByFile(file)
    },
  },
  "projectId": "",
  "env": {},
  "video": false,
  "videoUploadOnPasses": false,
  "defaultCommandTimeout": 8000,
  "pageLoadTimeout": 60000,
  "requestTimeout": 15000,
  "numTestsKeptInMemory": 1,
  "responseTimeout": 60000,
  "viewportWidth": 1440,
  "viewportHeight": 900,
  "waitForAnimations": true,
  "taskTimeout": 60000,
  "trashAssetsBeforeRuns": false,
  "experimentalWebKitSupport": true,
  "retries": {
    "runMode": 1,
    "openMode": 0
  },
  "screenshotsFolder": "src/screenshots",
});
```

### Desired behavior

The tests should run successfully on WebKit.

### Test code to reproduce

1. clone https://github.com/Light07/SafariProblemCypress.git
2. execute yarn
3. run "yarn debug"

### Cypress Version

10.8.0

### Node version

v16.17.0

### Operating System

MacOS M1

**Comment by ZachJW34** [comment]
Thanks for opening the issue @Light07, I pulled down the and I'm seeing the same issue. I was able to get it working by removing the `experimentalSessionAndOrigin: true,` flag as [experimentalWebKitSupport](https://docs.cypress.io/guides/guides/launching-browsers#WebKit-Experimental) does not support session and origin (yet).

If you rely on `experimentalSessionAndOrigin` for your test suite to pass, then you'll have to hold off on testing with Webkit until we add full support.

I'm going to keep this issue open and have it routed to the appropriate team as an error message would have been helpful in this situation.
